### DISCUSSION TITLE
Error importing result nodeset companion specs

### DISCUSSION THREAD

**Discussion by th3pr0gr4mm** [FreeOpcUa/opcua-asyncio#1361] [General]
Hey, I'm trying to import diffferent companion specs for a server. Had no problem till now with Di, Machinery, Padim and Irdi. However, when trying to import the machinery result though I am getting the error: "RuntimeError: Unknown datatype for field: StructureField". I think the error comes from the TrimmedString datatype. I also don't find it in the ObjectIds. Anyone has an idea how to fix it?

**Comment by GoetzGoerisch** [comment]
[TrimmedString ](https://reference.opcfoundation.org/Core/Part5/v105/docs/12.43)is part of UA 1.05.2
Make shure that this is loaded.

**Reply by schroeder-** [reply]
We are currently using UA 1.05.01 as Standard Addresspace, so this type is not in the addresspace.

**Reply by th3pr0gr4mm** [reply]
Oh okay so I just have to wait I guess? Do you know yet when you will update to 1.05.2?

**Comment by th3pr0gr4mm** [comment]
How can I load that one? I thought when importing companion specs I shouldn't import the base Opcua?
